storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1397,7 +1397,7 @@ backup_files(const char *from, bool prep_mode)
 			/* backup fake file into empty directory */
 			char path[FN_REFLEN];
 			ut_snprintf(path, sizeof(path),
-					"%s/db.opt", node.filepath);
+					"%sdb.opt", node.filepath);
```

Comment:
this is a bit risky. What if the path doesn't end with trailing slash? My suggestion is to keep this %s/db.opt thing.  Add code to remove trailing slash in path.